### PR TITLE
Add long diagnostics for E0062, E0063 and E0137

### PR DESCRIPTION
**Pull request by nham**
cc #24407

### CODE REVIEW

File: src/librustc_typeck/diagnostics.rs [hunk 1]
```diff
@@ -12,6 +12,16 @@
 
 register_long_diagnostics! {
 
+E0062: r##"
+This error indicates that during an attempt to build a struct or struct-like
+enum variant, one of the fields was specified more than once.
```
Comment (alexcrichton) [inline src/librustc_typeck/diagnostics.rs]: Perhaps this could elaborate in saying that each field must be specified exactly one time?

Comment (nham) [inline src/librustc_typeck/diagnostics.rs]: Thanks, that's a good suggestion. I've added this.

**Comment by nham** [comment]
rebased

**Pull request merged**